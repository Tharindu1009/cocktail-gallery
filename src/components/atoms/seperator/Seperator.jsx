import './styles/seperator.scss';
import * as React from 'react';

function Seperator() {
    return (
        <div data-testid="seperator" className='seperator'>
        </div>
    );
}

export default Seperator;
