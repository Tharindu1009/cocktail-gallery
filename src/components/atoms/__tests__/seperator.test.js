import { render, cleanup } from "@testing-library/react";
import TestRenderer from 'react-test-renderer';
import Seperator from "../seperator/Seperator";

describe('Button', () => {
    afterEach(() => {
        cleanup();
    })

    it('should render seperator component correctly', () => {
        const { getByTestId } = render(<Seperator />);
        const seperatorElement = getByTestId('seperator');
        expect(seperatorElement).toBeInTheDocument();
    });

    it('matches seperator snapshot', () => {
        const tree = TestRenderer.create(<Seperator />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});