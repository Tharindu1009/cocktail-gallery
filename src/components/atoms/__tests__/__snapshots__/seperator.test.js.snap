// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches seperator snapshot 1`] = `
<div
  className="seperator"
  data-testid="seperator"
/>
`;

exports[`matches seperator snapshot 1`] = `
<div
  className="seperator"
  data-testid="seperator"
/>
`;
